Libraries/Image/Image.ios.js

```diff
@@ -221,7 +221,7 @@ var Image = React.createClass({
       RawImage = RCTImageView;
     }
 
-    if (this.props.src) {
+    if (!this.props.source && this.props.src) {
```

Comment:
IMHO if someone does `<Image source={...} src={...} />` we should still warn, the code is not correct.